src/d_loop.c

```diff
@@ -674,8 +674,7 @@ static void SinglePlayerClear(ticcmd_set_t *set)
     }
 }
 
-void tick_sticky_msgs();
-
+void __attribute__((weak)) tick_sticky_msgs() {}
```

Comment:
Function is not defined for hexen. While unsupported afaik, it still produces compile errors.
Instead, define it as weak symbol that gets overwritten by doom and heretic, but is noop on hexen

Reply:
I never tried to compile Hexen. There is no AP support for it. But nice that you tried to make sure it still builds anyway :)
I never knew about `__attribute__((weak))`. Is this something supported by MSVC ?

Reply:
Not sure, and I dont have a msvc install to test. Could I ask you to test it?

Reply:
yup I will later today. Worse case we can #ifdef it WIN32 is the preprocessor used for windows.

Reply:
or you could just remove that change, as we dont try to compile strife and hexen. `make crispy-doom` and `make crispy-heretic` and `make crispy-setup` is all we care.

Reply:
Reverted this change